pages/plugin/[[...page]].tsx

```diff
@@ -29,15 +30,42 @@ function PluginLayout(props) {
   )
 }
 
-const { getStaticPaths, getStaticProps } = getStaticGenerationFunctions({
-  strategy: 'fs',
-  localContentDir: CONTENT_DIR,
-  navDataFile: NAV_DATA,
-  product: SOURCE_REPO,
-  githubFileUrl(filepath) {
-    return `https://github.com/hashicorp/${SOURCE_REPO}/blob/${DEFAULT_BRANCH}/${filepath}`
-  },
-})
+const { getStaticPaths: _getStaticPaths, getStaticProps } =
+  getStaticGenerationFunctions({
+    strategy: 'fs',
+    localContentDir: CONTENT_DIR,
+    navDataFile: NAV_DATA,
+    product: SOURCE_REPO,
+    githubFileUrl(filepath) {
+      return `https://github.com/hashicorp/${SOURCE_REPO}/blob/${DEFAULT_BRANCH}/${filepath}`
+    },
+  })
+
+/**
+ * This is a temporary hack to allow multiple `plugin` page components that
+ * may generate duplicate static paths. When `plugin` content is split into
+ * terraform-docs-common, this can be removed.
+ *
+ * - https://nextjs.org/docs/messages/conflicting-ssg-paths
+ */
+const getStaticPaths = async (ctx) => {
+  const res = await _getStaticPaths(ctx)
+
+  // remove paths for "framework", "log", "mux", and "sdkv2"
```

Comment:
👍 this is neat! i have a vague familiarity with this approach since I think we've done something pretty similar for the new `.io/docs` landing pages. Seems great to me as an interim solution 💯